### PR TITLE
Add payment service

### PR DESCRIPTION
**Pull request by Yuruch**

### CODE REVIEW

**Review by Irina17191 — APPROVED** [review]

File: payment_service/admin.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```
Comment (necobra) [inline payment_service/admin.py]: just register the payment model to admin panel

File: payment_service/views.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    serializer_class = PaymentSerializer
+
+    def get_queryset(self):
+        user = self.request.user
+        if user.is_staff:
+            return Payment.objects.all()
+        else:
+            return Payment.objects.filter(borrowing_id__in=user.borrowings)
+
+
```
Comment (necobra) [inline payment_service/views.py:80]: we can decorate func based view with @api_view(["GET", "POST"]) for clearer definition and automatically drop unnecessary methods

File: payment_service/views.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+                    request, "payment/success.html", {"payment": payment}
+                )
+            else:
+                return HttpResponse("Payment was not successful", status=400)
+        except Payment.DoesNotExist:
+            return HttpResponse("Payment not found", status=404)
+
+    return HttpResponse("Session ID not provided", status=400)
+
+
```
Comment (necobra) [inline payment_service/views.py:142]: the same here

File: borrowings_service/views.py [hunk 4]
```diff
@@ -74,5 +114,27 @@ def return_borrowing(self, request, pk=None):
 
         borrowing.book.inventory += 1
         borrowing.book.save()
-
+        actual_date = borrowing.actual_return_date
+
+        # if actual_date > expected_date:
```
Comment (necobra) [inline borrowings_service/views.py]: ??

**Review by necobra — APPROVED** [review]

**Review by necobra — APPROVED** [review]

**Pull request merged**